worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/GriefDefenderFeature.java

```diff
@@ -25,21 +23,15 @@ public GriefDefenderFeature(final Plugin GriefDefenderPlugin) {
     }
 
     public boolean isAllowed(Player player, Claim claim, MaskType type) {
-        return GriefDefender.getCore().isEnabled(player.getWorld().getUID()) && !claim.isWilderness() && (claim
-                .getOwnerName()
-                .equalsIgnoreCase(player.getName()) || claim.getOwnerUniqueId().equals(player.getUniqueId()) ||
-                type == MaskType.MEMBER && claim.getUserTrusts(TrustTypes.BUILDER).contains(player.getUniqueId()));
+        return type == MaskType.MEMBER && claim.isUserTrusted(player.getUniqueId(), TrustTypes.BUILDER);
     }
 
     @Override
     public FaweMask getMask(final com.sk89q.worldedit.entity.Player wePlayer, MaskType type, boolean isWhitelist) {
         final Player player = BukkitAdapter.adapt(wePlayer);
-        final Location loc = player.getLocation();
-        final Vector3i vector = Vector3i.from(loc.getBlockX(), loc.getBlockY(), loc.getBlockZ());
-        final Claim claim = GriefDefender.getCore().getClaimManager(loc.getWorld().getUID()).getClaimAt(vector);
-        if (!claim.isWilderness()) {
+        Claim claim = GriefDefender.getCore().getClaimAt(player.getLocation());
+        if (claim != null && !claim.isWilderness()) {
             if (isAllowed(player, claim, type)) {
-                claim.getGreaterBoundaryCorner().getX();
```

Comment:
Which versions of GD did the old diff used to support and which versions does the new one support?

Reply:
Now it support 2.0.0 and newer, I haven't tested it on 1.5.10 but may works.

Reply:
Good to know, I'll update the wiki instructions once merged.